src/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -78,6 +81,14 @@ public String getResultAlias() {
 	}
 
 	/**
+	 * The useObjDomainAsKey property allows you to specify the use of the Domain portion of the Object Name
+	 * as part of the output key instead of using the ClassName of the MBean which is the default behavior.
+	 */
+	public boolean isUseObjDomainAsKey() {
```

Comment:
You are right. `isUsingObjDomainAsKey` is better English.

However in the JSON, `useObjDomainAsKey` is more correct, it wouldn't sound valid to say `using` there (also I shortened Object to Obj to match the existing `obj` parameter and make it clear that this property is referring to the same thing).

While `isUseObjDomainAsKey` isn't great English I thought it was clearer in the code to have the method name match the attribute it is returning exactly, and to follow the existing convention in the class to return boolean attributes using an `is` prefix, such as `isAllowDottedKeys`. I think it's a reasonable compromise and OK for it not to be perfect English in the code.

That said, I don't have a strong opinion on it and am happy to change it if you prefer.